internal/services/springcloud/spring_cloud_java_deployment_resource.go

```diff
@@ -356,31 +356,17 @@ func resourceSprintCloudJavaDeploymentSchema() map[string]*pluginsdk.Schema {
 						Type:     pluginsdk.TypeString,
 						Optional: true,
 						Computed: true,
-						ValidateFunc: validation.StringInSlice([]string{
-							"500m",
-							"1",
-							"2",
-							"3",
-							"4",
-						}, false),
+						// There are additional options when the service team allow them
```

Comment:
```suggestion
							// NOTE: we're intentionally not validating this field since additional values are possible when enabled by the service team
```